packages/admin/src/table/filterbar/FilterBarActiveFilterBadge.tsx

```diff
@@ -0,0 +1,60 @@
+import { createStyles, Theme, Typography, WithStyles, withStyles } from "@material-ui/core";
+import * as React from "react";
+
+export type CometAdminFilterBarActiveFilterBadgeClassKeys = "hasValueCount";
+
+export const dirtyFieldsCount = (values: Record<string, any>) => {
+    return Object.values(values).reduce((acc, val) => {
+        if (val) {
+            if (Array.isArray(val)) {
+                acc += val.length;
+            }
+            acc++;
+        }
+        return acc;
+    }, 0);
+    /*
+    TODO: what is the use case of this?
```

Comment:
Please either remove this code or use it. (if removing, also remove the difference helper method)

we can also re-add it alter when needed.